tests/test_document_receipt_replay_guard.py

```
@@ -0,0 +1,86 @@
+
+from korgan import document_receipt_replay_guard as guard
+
+
+class FakePool:
+    def __init__(self, *, insert_result: str, row=None) -> None:
+        self.insert_result = insert_result
+        self.row = row
+        self.executed = 0
+        self.fetches = 0
+
+    async def execute(self, *_args, **_kwargs) -> str:
```

Ruff (0.16.2)

[warning] Missing type annotation for `*_args`

(ANN002)